spec/models/tslr_claim_spec.rb

```diff
@@ -134,6 +145,31 @@
     end
   end
 
+  context "when saving in the “bank-details” validation context" do
+    it "validates that the bank_account_number and bank_sort_code are present" do
+      expect(TslrClaim.new).not_to be_valid(:"bank-details")
+      expect(TslrClaim.new(bank_sort_code: "12 34 56", bank_account_number: "87 65 43 21")).to be_valid(:"bank-details")
+    end
+  end
+
+  describe "#bank_account_number" do
```

Comment:
As discussed, the two specs in these describe blocks are really testing the before save normalisation and not the attribute reader, so more accurate to move the `it` blocks up into the `save` context at the top and ditch the extra `describes.

Reply:
I've gone ahead and re-factored the TSLRClaim spec as we discussed, as well as moving the `it` blocks mentioned above, felt like it was a natural time to pick up on it.